### PR TITLE
Issue/3668 rotate wpadmin

### LINKED ISSUE #3668: View Admin: In-app browser refreshes view after rotating device
When you use the in-app browser to view a site's wp admin dashboard, any time you rotate the device it refreshes and goes back to the main dashboard screen at `/wp-admin`. This loses your place and loses any unsaved work.

Steps to reproduce:

1. Select a site in the app and select "View Admin"
2. In the wp admin dashboard, navigate to another part of the dashboard (e.g. Settings > General)
3. Rotate your device

Result: You are returned to the main dashboard screen (instead of staying in the part of the dashboard you had navigated to).

Reported in an app review; tested and reproduced on Nexus 9, Android 6.0.1, app beta v. 5.0-rc-2

### PR DESCRIPTION
**Pull request by nbradbury**
Resolved #3668 by:

1. Having WebViewActivity save & restore the webView state, which retains the current URL
2. Only loading the desired URL on initial creation (ie: when `savedInstanceState` is null) so it doesn't reload upon rotation

Note that this problem wasn't specific to "View Admin" - it happened everywhere a WebViewActivity was used.

### CODE REVIEW

**Comment by maxme** [comment]
Looks good, but there is something weird on Calypso pages, after rotation the screen is stuck on the empty page logo (loading).

screen shot 2016-01-29 at 13 59 53

**Comment by nbradbury** [comment]
@maxme Glad you caught that problem reloading Calypso pages! Turns out that even when restoring the webView state, the previous settings (incuding WebViewClient and WebChromeClient) still need to be applied. So, I updated WPWebViewActivity with the necessary corrections.

These corrections required updating SSLCertsViewActivity and LicensesActivity, since they implement WPWebViewActivity.

In addition, I noticed that "View Site" suffered from the same loss of navigation after rotation, so I updated it to implement WPWebViewActivity.

TL;DR Ready for another look.

**Comment by maxme** [comment]
Looks good! :shipit:
